src/ImageSharp/Formats/Webp/Lossy/Vp8Histogram.cs

```diff
@@ -52,11 +61,38 @@ public void CollectHistogram(Span<byte> reference, Span<byte> pred, int startBlo
                 Vp8Encoding.FTransform(reference.Slice(WebpLookupTables.Vp8DspScan[j]), pred.Slice(WebpLookupTables.Vp8DspScan[j]), this.output, this.scratch);
 
                 // Convert coefficients to bin.
-                for (int k = 0; k < 16; ++k)
+#if SUPPORTS_RUNTIME_INTRINSICS
+                if (Avx2.IsSupported)
                 {
-                    int v = Math.Abs(this.output[k]) >> 3;
-                    int clippedValue = ClipMax(v, MaxCoeffThresh);
-                    ++this.distribution[clippedValue];
+                    // Load.
+                    ref short outputRef = ref MemoryMarshal.GetReference<short>(this.output);
```

Comment:
```suggestion
                    ref short outputRef = ref MemoryMarshal.GetArrayDataReference(this.output);
```
[Avoids](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAJI8YUCAAcAymIBuvMDFwBuGjWIBmJqQYBhBgG8aDEwwlRes7BhgNY2ACYQuAGwCeDXAAtoGANoAugwQHBgSoQwAvAxcMADunj5Q/uRIAarUpgzGppp2MABmib4MALLkABQAlDkmRplZprBF3iUhYaFshVH5RaUw+NBupdhQ3tguLADiMBhdBWIwXIoAPK3JAHwVGF68uCzt4RhVGY25AOy9waFH86emAL612Q0mec3FyWWk1c/1ZyYPusMNcOnNutEPv1BlBhqNxpMZhgAIJQKDYNwAEWs2Hmi2WMG2u32h1CJ2eWWIlw+pPBBXuJie1AeQA==) the intermediate `array -> Span` step.

But this API is only [available from .NET 5 onwards](https://apisof.net/catalog/c8cb970a-e493-764c-7ad9-3aadb89e1d89) 😢 

So maybe just leave the code as is, and remember this API for other uses 😉

Reply:
We should definitely start using this once we get rid of supporting the older frameworks